gems/aws-sdk-core/spec/seahorse/client/handler_builder_spec.rb

```diff
@@ -48,7 +48,7 @@ def initialize
           expect(handler_class.new.call('context')).to eq('context')
         end
 
-        it 'accepts a step with the block' do
+        it 'accepts a step with the block', rbs_test: :skip do
```

Comment:
Why do these tests need to be skipped for rbs?

Reply:
I have not been able to investigate the details, but it seems to be a bug on the rbs side that prevents testing 😢

```
$ RUBYOPT="-r bundler/setup -r rbs/test/setup" RBS_TEST_RAISE=true RBS_TEST_LOGLEVEL=error RBS_TEST_OPT="-I gems/aws-sdk-core/sig -I gems/aws-sdk-core/sig" RBS_TEST_TARGET="Aws,Aws::*,Seahorse,Seahorse::*" bundle exec rspec gems/aws-sdk-core/spec/seahorse/client/handler_builder_spec.rb
...F..........

Failures:

  1) Seahorse::Client::HandlerBuilder#handler accepts a step with the block
     Failure/Error: super(context)

     NoMethodError:
       super called outside of method
     # ./gems/aws-sdk-core/spec/seahorse/client/handler_builder_spec.rb:54:in `block (4 levels) in <module:Client>'
     # /Users/ksss/src/github.com/ksss/rbs/lib/rbs/test/hook.rb:113:in `instance_exec'
     # /Users/ksss/src/github.com/ksss/rbs/lib/rbs/test/hook.rb:113:in `block in handler__with__RBS_TEST_a0f0b5_878362e7'
     # ./gems/aws-sdk-core/spec/seahorse/client/handler_builder_spec.rb:68:in `block (3 levels) in <module:Client>'

Finished in 0.2305 seconds (files took 5.46 seconds to load)
14 examples, 1 failure

Failed examples:

rspec ./gems/aws-sdk-core/spec/seahorse/client/handler_builder_spec.rb:51 # Seahorse::Client::HandlerBuilder#handler accepts a step with the block
```